build.gradle

```diff
@@ -161,7 +161,7 @@ task(loadSchoolDataForConformance, type: Exec) {
 }
 
 task createConformanceTestVenv(type: Exec) {
-    commandLine 'pyvenv-3.5', '.venv'
+    commandLine 'pyvenv', '.venv'
```

Comment:
Does this ensure python3 is used? Or, does this prevent using python2?

Reply:
Looks like pyvenv is deprecated anyway: https://docs.python.org/3/library/venv.html
It says we should be using `python -m venv`, which doesn't seem to work with python 2.

Would mandating python3 be acceptable?

Reply:
I think it is acceptable, yes. @gidsg @michaelabenyohai ?

Reply:
@maxf you can try this although we have had problems with this failing on Travis CI when we attempted to switch from `pyvenv` currently we have this condition to use an old travis build which does not come bundled with Python 3: https://github.com/openregister/openregister-java/blob/086128b2b0ea8fd07db8d5dd2dcdecc410c60ffe/.travis.yml#L1

My suggestion would be to merge this as-is as it is an improvement on what we have and then in a separate PR look to update the travis build and use python3 `venv`.

Reply:
fine by me!